### ISSUE TITLE
Use Debian shellinabox (aka Webshell) package (rather than our own fork)

### ISSUE THREAD

**Issue by JedMeister** [turnkeylinux/tracker#918]
I'm pushing forward ahead on this but thought I'd note it here for discussion.

For those that are unaware of the history:

We've been packaging shellinabox since TurnKey started (or very soon after). Although it was abandoned by the original dev some time ago. We've added a couple of patches, but otherwise it's still running the old abandoned source.

The original code was hosted on [Google Code](https://code.google.com/archive/p/shellinabox/downloads) and the last "official" release was in 2012. Someone forked it and uploaded to [GitHub](https://github.com/shellinabox/shellinabox) and has continued with development. It's now in the [Debian repos](https://packages.debian.org/stretch/shellinabox).

**Comment by JedMeister** [comment]
This is already implemented in the stretch-dev common branch. Leaving open for now, but we'll push ahead.

**Comment by JedMeister** [comment]
I've changed my mind and I'm going to start closing v15.0 issues as they're merged with the relevant TurnKey repos. 

In the case of [common](https://github.com/turnkeylinux/common), I had been holding off, awaiting the merging of changes into master. But I've decided that so long as the code has been merged with the [stretch-dev](https://github.com/turnkeylinux/common/tree/stretch-dev) branch, that will do (and we'll merge stretch-dev into master some time soon).

FWIW this particular issue doesn't actually require anything other than for us to not provide a `shellinabox` package in our repo. So long as we don't provide a package, the install process will simply use the one from the Debian repos. :smile:
